doc/source/whatsnew/v1.0.0.rst

```diff
@@ -824,6 +824,7 @@ Plotting
 - :func:`set_option` now validates that the plot backend provided to ``'plotting.backend'`` implements the backend when the option is set, rather than when a plot is created (:issue:`28163`)
 - :meth:`DataFrame.plot` now allow a ``backend`` keyword arugment to allow changing between backends in one session (:issue:`28619`).
 - Bug in color validation incorrectly raising for non-color styles (:issue:`29122`).
+- Allow :meth: `DataFrame.plot` to plot ``objects`` and ``datetime`` type data (:issue:`18755`, :issue:`30391`)
```

Comment:
this is more specific yes? it only applies to scatter right

Reply:
thanks for taking a look! @jreback indeed, should have been more specific.

Updated.